config/database.yml

```diff
@@ -20,6 +20,9 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: 5
+  username: postgres
+  password: password
+  host: localhost
```

Comment:
`database.yml` has been added to `.gitignore` and the former has become instead `database.yml.example`, per Jon's advice